app/menus/CollectionMenu.js

```diff
@@ -114,7 +114,9 @@ class CollectionMenu extends React.Component<Props> {
             ref={(ref) => (this.file = ref)}
             onChange={this.onFilePicked}
             onClick={(ev) => ev.stopPropagation()}
-            accept="text/markdown, text/plain"
+            accept={["text/markdown", "text/plain"]
```

Comment:
Just noticed in testing there's another spot that needs updating in `DropToImport` component, so best to make this available directly on the store so we can use it twice - `importFiletypesCSV` or something.

Reply:
https://github.com/outline/outline/pull/1469/commits/44a861b3af5b5db96d9a0a15c042bda85c2f2b17#diff-5dead950e66336055adf3edbd1311562R117

Reply:
This one was only partly addressed, the `DropToImport` component still not accepting docx types